scripts/build-process/generate-pages-from-markdown.js

```diff
@@ -106,13 +107,20 @@ const enrichedMarkdownFiles = markdownFiles.map(fileName => {
 // Create the routes that'll be used in 'docs/src/routes/generated/routes.js'.
 const routes = {};
 enrichedMarkdownFiles.forEach(
-  ({ category, path, content, title, subtitle }) => {
+  ({ category, fileName, path, content, title, subtitle }) => {
     if (!routes[category]) {
       routes[category] = [];
     }
 
     // Escape backticks to prevent them terminating the string.
-    const escapedContent = content.replace(/`/g, '\\`');
+    let escapedContent = content.replace(/`/g, '\\`');
+
+    // Add a footer to the bottom of the content for editing.
+    escapedContent += `
+## Improve this page
+
+[Edit this page on GitHub](https://github.com/skyscanner/backpack-docs/edit/master/${PATH_TO_MARKDOWN_FILES}/${fileName})
+    `;
```

Comment:
I feel like this should maybe be part of the MarkdownPageRenderer rather than just shoving on some extra MD. I guess both are valid though.

What happens if the MD file doesn't end with a line break?

Reply:
Yeah we could potentially do that, I just felt like it was better in here because we have access to the filename more easily, and if a page is autogenerated we **know** it can be edited through this process, whereas the MarkdownPage component is agnostic, it just displays whatever its given.

> What happens if the MD file doesn't end with a line break?

I'll check and fix if any issues

Reply:
Seems to be fine